app/src/main/res/layout-land/submitted_answer_item.xml

```diff
@@ -84,9 +85,15 @@
       android:src="@drawable/ic_check_primary"
       android:visibility="@{viewModel.isCorrectAnswer &amp;&amp; !viewModel.hasConversationView ? View.VISIBLE : View.GONE}"
       app:layout_constraintBottom_toBottomOf="parent"
-      app:layout_constraintEnd_toEndOf="parent"
-      app:layout_constraintHorizontal_bias="0.0"
-      app:layout_constraintStart_toEndOf="@+id/submitted_answer_container"
+      app:layout_constraintStart_toEndOf="@+id/submitted_answer_text_view"
       app:layout_constraintTop_toTopOf="parent" />
-  </LinearLayout>
+
+    <androidx.constraintlayout.widget.Guideline
+      android:id="@+id/answer_tick_guideline"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      app:layout_constraintGuide_end="@{@dimen/answer_tick_margin + @dimen/answer_tick_size}" />
```

Comment:
Thanks for updating these dimensions--the ``answer_tick`` ImageView should be updated to use it for its width/height & margin (otherwise there's no benefit in having shared dimensions).

Reply:
Done